2019-12-04 20:05:51,918 ERROR: Exception on /messages [GET] [in /home/donjoe/Desktop/projects/sched/venv/lib/python3.5/site-packages/flask/app.py:1891]
Traceback (most recent call last):
  File "/home/donjoe/Desktop/projects/sched/venv/lib/python3.5/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/donjoe/Desktop/projects/sched/venv/lib/python3.5/site-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/donjoe/Desktop/projects/sched/venv/lib/python3.5/site-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/donjoe/Desktop/projects/sched/venv/lib/python3.5/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/donjoe/Desktop/projects/sched/venv/lib/python3.5/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/donjoe/Desktop/projects/sched/venv/lib/python3.5/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/donjoe/Desktop/projects/sched/venv/lib/python3.5/site-packages/flask_login/utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "/home/donjoe/Desktop/projects/sched_blog/app/main/routes.py", line 199, in messages
    next_url=next_url, prev_url=prev_url)
  File "/home/donjoe/Desktop/projects/sched/venv/lib/python3.5/site-packages/flask/templating.py", line 140, in render_template
    ctx.app,
  File "/home/donjoe/Desktop/projects/sched/venv/lib/python3.5/site-packages/flask/templating.py", line 120, in _render
    rv = template.render(context)
  File "/home/donjoe/Desktop/projects/sched/venv/lib/python3.5/site-packages/jinja2/environment.py", line 1008, in render
    return self.environment.handle_exception(exc_info, True)
  File "/home/donjoe/Desktop/projects/sched/venv/lib/python3.5/site-packages/jinja2/environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/donjoe/Desktop/projects/sched/venv/lib/python3.5/site-packages/jinja2/_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "/home/donjoe/Desktop/projects/sched_blog/app/templates/messages.html", line 1, in top-level template code
    {% extends "base.html" %}
  File "/home/donjoe/Desktop/projects/sched_blog/app/templates/base.html", line 23, in top-level template code
    {% block content %} {% endblock %}
  File "/home/donjoe/Desktop/projects/sched_blog/app/templates/messages.html", line 6, in block "content"
    {% include '_post.html' %}
  File "/home/donjoe/Desktop/projects/sched/venv/lib/python3.5/site-packages/flask/templating.py", line 60, in get_source
    return self._get_source_fast(environment, template)
  File "/home/donjoe/Desktop/projects/sched/venv/lib/python3.5/site-packages/flask/templating.py", line 89, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: _post.html
2019-12-04 20:09:11,658 INFO: Microblog startup [in /home/donjoe/Desktop/projects/sched_blog/app/__init__.py:71]
2019-12-04 20:09:11,659 INFO: Microblog startup [in /home/donjoe/Desktop/projects/sched_blog/app/__init__.py:89]
2019-12-04 20:09:11,659 INFO: Microblog startup [in /home/donjoe/Desktop/projects/sched_blog/app/__init__.py:89]
2019-12-04 20:34:14,290 INFO: Microblog startup [in /home/donjoe/Desktop/projects/sched_blog/app/__init__.py:71]
2019-12-04 20:34:14,290 INFO: Microblog startup [in /home/donjoe/Desktop/projects/sched_blog/app/__init__.py:89]
2019-12-04 20:34:14,290 INFO: Microblog startup [in /home/donjoe/Desktop/projects/sched_blog/app/__init__.py:89]
2019-12-04 20:48:03,662 INFO: Microblog startup [in /home/donjoe/Desktop/projects/sched_blog/app/__init__.py:71]
2019-12-04 20:48:03,663 INFO: Microblog startup [in /home/donjoe/Desktop/projects/sched_blog/app/__init__.py:89]
2019-12-04 20:48:03,663 INFO: Microblog startup [in /home/donjoe/Desktop/projects/sched_blog/app/__init__.py:89]
2019-12-04 20:55:48,971 INFO: Microblog startup [in /home/donjoe/Desktop/projects/sched_blog/app/__init__.py:71]
2019-12-04 20:55:48,971 INFO: Microblog startup [in /home/donjoe/Desktop/projects/sched_blog/app/__init__.py:89]
2019-12-04 20:55:48,971 INFO: Microblog startup [in /home/donjoe/Desktop/projects/sched_blog/app/__init__.py:89]
